src/App.js

```diff
@@ -1,10 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import data from './data/dummy.json';
 
 const App = () => {
+  const [response, setResponse] = useState({});
+  const [score, setScore] = useState(0);
+  const [displayScore, setDisplayScore] = useState(false);
+
+  const handleResponse = e => {
+    const selectedAnswer = { [e.target.name]: e.target.value };
+    setResponse({ ...response, ...selectedAnswer });
+  };
+
+  const checkAnswers = e => {
+    e.preventDefault();
+    const handleScore = data
```

Comment:
`handle` is usually used for event listener callbacks, which is not the case here. `handle` is also a bit generic, and here I think we could have a more descriptive name like `calculateScore`